### PR TITLE
Cross platform file picker

### PR DESCRIPTION
**Pull request by wellhat**

### CODE REVIEW

**Review by tomShoutTheSecond — CHANGES_REQUESTED** [review]
Two issues I found on this branch

1. Animation glitch: AnimateSlideSendButton should be called immediately after the user selects a file (ie. before the upload starts)
2. Choosing "Select Image" takes me to Android's file browser instead of my default gallery app

**Comment by wellhat** [comment]
Updated @tomShoutTheSecond

File: src/SkyDrop.Core/ViewModels/DropViewModel.cs [hunk 1]
```diff
@@ -135,23 +140,55 @@ private async Task StartSendFile()
...
+                    Log.Trace("No file was picked.");
+                    ResetUI();
+
+                    return;
+                }
+
+                var firstFile = pickedFiles.First();
+
+                byte[] fileBytes = null;
+                using (var stream = await firstFile.OpenReadAsync())
```
Comment (tomShoutTheSecond) [inline src/SkyDrop.Core/ViewModels/DropViewModel.cs:176]: NullReferenceException thrown here when user presses hardware back button while browsing for a file

**Comment by tomShoutTheSecond** [comment]
Slide animation glitchy

https://user-images.githubusercontent.com/42177014/113516011-e2911700-956f-11eb-9096-993356c73518.mp4

**Comment by tomShoutTheSecond** [comment]
Bug: send file button does not work while the barcode is loading (might also be an issue on main branch)

**Pull request merged**